joern-cli/frontends/gosrc2cpg/src/main/scala/io/joern/gosrc2cpg/astcreation/AstForPrimitivesCreator.scala

```diff
@@ -27,4 +30,15 @@ trait AstForPrimitivesCreator { this: AstCreator =>
     }
   }
 
+  protected def getTypeOfToken(basicLit: ParserNodeInfo): String = {
+    Try(basicLit.json(ParserKeys.Kind).str match {
+      case "INT"    => "int"
```

Comment:
I suppose one day we may want to prefix these with `__builtin.` or something, but for this PR this is good

Reply:
@khemrajrathore we should add the TODO: for the points mentioned by the @DavidBakerEffendi